src/mumble/Overlay.cpp

```diff
@@ -212,7 +212,13 @@ Overlay::~Overlay() {
 }
 
 void Overlay::setActive(bool act) {
+	if (!act && !m_initialized.load()) {
+		// Disabling when the Overlay hasn't been initialized yet, doesn't make much sense
+		return;
+	}
+
 	if (!m_initialized.load()) {
+		qDebug() << "Overlay init";
```

Comment:
**issue:** We use a `Overlay: ` prefix by convention in this file to make the debug messages consistent and clearer/scannable.

```suggestion
		qDebug() << "Overlay: Initializing...";
```

(Does applying GitHub suggestions work without line ending changes being produced? I had bad experiences with GitLab concerning that. Looks like the file is LF [already].)

Reply:
Squashing the change into the actual commit would be better anyway… Applying the suggestion will produce an additional commit.

Reply:
This log shouldn't actually be part of the final change. I introduced it for debugging purposes only. I'll remove it ^^